index.html

```diff
@@ -0,0 +1,123 @@
+                            <div class="form-group">
+                                <!-- <label for="formGroupExampleInput2">Author</label> -->
+                                <input type="text" name="author" class="form-control" id="bookAuthor" placeholder="What's the author name?" required>
+                            </div>
+
+                            <div class="form-group">
+                                <!-- <label for="formGroupExampleInput">Number of Page(s)</label> -->
+                                <input type="number" name="pages" class="form-control" id="bookPage" placeholder="Enter the book's page" required>
+                            </div>
+                         
+
+                            <div class="form-check">
+                                <input class="form-check-input" name="read" type="checkbox" value="off" id="bookRead" >
+                                <label class="form-check-label" for="gridRadios2">
+                                  Read
+                                </label>
+                            </div>
+                            <br>
+                            <div class="form-group">
+
+                                <button class="btn btn-success " id="addBookButton"> Add New Book</button>
+                                <button type="reset" class="btn btn-light " id="addBookButton" id="cancelFormContainer" onclick="toggleFormContainer()" > Cancel</button>
+                            </div>
+                        </form>
+                    </div>
+                </div>
+                <div class="card">
+                    <div class="">
+
+
+                        <table class="table table-borderless table-hover" id="table-data">
+                            <thead class="thead-dark">
+                                <tr>
+                                    <th scope="col" width="50%">Title</th>
+                                    <th scope="col">Author</th>
+                                    <th scope="col">Pages</th>
+                                    <th scope="col">Read</th>
+                                    <th scope="col">Actions</th>
+                                </tr>
+                            </thead>
+                            <tbody>
+                                <!-- <tr>
+                                    <td></td>
+                                    <td></td>
+                                    <td></td>
+                                    <td>
+                                        <div class="switch" >
+                                            <div class="cursor"></div>
+                                        </div>
+                                    </td>
+                                    <td>
+                                        <div class="btn btn-danger btn-sm" id="removeBook">Remove</div>
+                                    </td>
+                                </tr> -->
```

Comment:
- [ ] Kindly remove this commented code. It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)).